### DISCUSSION TITLE
Sort and Finalize the result?

### DISCUSSION THREAD

**Discussion by HocheolJang426** [aertslab/SCENIC#278] [R] [answered]
Hi SCENIC user!

I'm wondering how to sort and finalize the result.
in output folder, MotifEnrichment_preview.html made, but this file is too complicated
NES are not arranged, and many of TF_highConf overlapped
 I also checked int folder, motifEnrichment_selfMotifs_wGenes.Rds, also this data.frame is complicated

I want to export my result like;
motif logo ; NES ; TF name ; Target gene
or any other good way  to visualize the SCENIC result/output

Is there a good way to visualize the result?

**Comment by s-aibar** [comment, marked as answer]
Duplicate to #279 and #280
